### ISSUE TITLE
convert eclipse C project to cmake project

### ISSUE THREAD

**Issue by mickae1** [15knots/cmake4eclipse#136]
Hi,

Is it possible to use this tools to convert a C project in eclipse into a cmake project ?

**Comment by emweber** [comment]
What kind of _cmake project_ are you referring to here?

Please ask questions like this on the mailing list.

**Comment by mickae1** [comment]
I wanted to say CMakeLists.txt
File

Le mar. 26 mai 2020 à 15:09, Martin Weber <notifications@github.com> a
écrit :

**Comment by emweber** [comment]
No, it cannot convert a C project in eclipse into a CMakeLists.txt file.

**Comment by mickae1** [comment]
Well,that would be a good feature :)
